README.md

```diff
@@ -1 +1,19 @@
-# wamr-app-framework
\ No newline at end of file
+# wamr-app-framework
```

Comment:
Can we add some introduction in the README? Refer to introduction in WAMR README:


Useful components and tools for building real solutions with WAMR vmcore:
[App-framework](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/app-framework/README.md): A framework for supporting APIs for the Wasm applications
[App-manager](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/app-mgr/README.md): a framework for dynamical loading the Wasm module remotely
[WAMR-IDE](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/test-tools/wamr-ide): An experimental VSCode extension for developing WebAssembly applications with C/C++

Reply:
Done